src/core_plugins/kibana/public/management/sections/indices/create_index_pattern_wizard/lib/__tests__/is_query_a_match.test.js

```diff
@@ -0,0 +1,31 @@
+import { isQueryAMatch } from '../is_query_a_match';
+
+describe('isQueryAMatch', () => {
+  it('should handle straight up matches', () => {
```

Comment:
"Handles" is a bit ambiguous to me. Can we rephrase/restructure these statements to assert the nature of the outcome? I'd suggest:

```js
describe('returns true', () => {
  it('for an exact value')
  it('for a pattern with a trailing wildcard')
  it('for a pattern with a leading wildcard')
  it('for a pattern with a middle and trailing wildcard')
  it('for a pattern with a middle wildcard')
  it(`for a pattern that's only a wildcard`)
});

describe('returns false', () => {
  it('for a non-matching pattern')
});
```

Reply:
Great!